package.json

```diff
@@ -29,50 +32,45 @@
     "prepublish": "yarn run build"
   },
   "dependencies": {
-    "babel-plugin-react-flow-props-to-prop-types": "0.15.0",
-    "babel-runtime": "6.26.0",
-    "invariant": "2.2.2",
-    "memoize-one": "3.0.1",
-    "prop-types": "15.5.10",
```

Comment:
I don't think we can remove prop-types, it's still used in some places... (typing context types for instance)

Reply:
Correct. I have re-added it